lib/screens/home_screen.dart

```diff
@@ -102,18 +96,19 @@ class HomeScreen extends StatelessWidget {
                         return Scaffold(
                           appBar: AppBar(title: Text('Course Directory')),
                           body: Center(
-                            child: FutureBuilder<StudyModule>(
-                              future: _getTestCourseDirectory(),
+                            child: FutureBuilder<List<StudyModule>>(
+                              future: fetchRootStudyModules(null),
```

Comment:
Please see the [service class](https://github.com/memoriesadrift/we-find/blob/main/lib/service/ufind_service.dart#L16) and the changes I made there. Instead of passing `null` as an argument, the same effect can be achieved by using an optional argument.

Reply:
I indeed forgot to change this. I will look into this at around lunch time